src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,81 @@
+package duke;
+
+import duke.command.Parser;
+import duke.exception.DukeException;
+import duke.exception.DukeIoException;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.stage.Stage;
```

Comment:
```suggestion
import duke.command.Parser;
import duke.exception.DukeException;
import duke.exception.DukeIoException;
import duke.task.TaskList;
import duke.ui.Ui;

import javafx.application.Application;
import javafx.application.Platform;
import javafx.stage.Stage;
```
As there are many imports here, I feel that a line break will make it easier to browse the list.

Reply:
Hi, while I agree with you on the neatness, doing so would actually fail CheckStyle based on the given coding standard configuration.

See error message below:
![image](https://user-images.githubusercontent.com/13163634/152351934-0727ff7c-8065-44cf-88f7-47e5c8bd9539.png)